webrtc.html

```diff
@@ -574,6 +574,15 @@
         following steps:</p>
         <ol>
           <li>
+            <p>Let <var>connection</var> be the current <code>
+            <a>RTCPeerConnection</a></code> object.</p>
+          </li>
+          <li>
+            <p>If <var>connection</var>'s [[<a>isClosed</a>]] slot is
+            <code>true</code>, return a promise rejected with an
+            <code>InvalidStateError</code>.</p>
```

Comment:
I notice we're inconsistent in this spec in requiring ", and abort these steps" after `return` and `throw` statements. Do we want to clean that up separately?